### DISCUSSION TITLE
Sveltekit

### DISCUSSION THREAD

**Discussion by Xasanboy-Dev** [community/community#44923] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello every one.
I have got one problem that, I can't use the [_**svelte-navigator**_](https://madewithsvelte.com/svelte-navigator)  with  [**svltekit**](https://kit.svelte.dev/)?
If you know answer to this question please help me.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by jge162** [comment]
@Xasanboy-Dev You will need to install the svelte-navigator package by running npm install svelte-navigator or yarn add svelte-navigator.

Next, you will need to configure Sveltekit to use svelte-navigator as the routing library. One way to do this is to import the svelte-navigator module in your src/main.js file and use it to configure the router.


```import { navigate } from 'svelte-navigator';

const routes = {
    '/': 'src/routes/home.svelte',
    '/about': 'src/routes/about.svelte',
};

navigate(routes);

Then, in your Svelte components, you can use the link and route components provided by svelte-navigator to handle routing.

<link to="/" >Home</link>
<link to="/about" >About</link>


Also you need to make sure that you are using the same version of svelte-navigator and sveltekit.

It's important to mention that Sveltekit comes with a built-in routing system, you may want to consider using that instead of svelte-navigator, it might be simpler to use and could save you some configuration time. Hope this answers your questions...
